### PR TITLE
Add the ability to add people via the channels modal - ZUR-168

### PR DESCRIPTION
**Pull request by OkoloJohnbosco**
**Before**

https://user-images.githubusercontent.com/47562075/201494878-83f88f69-db63-49ae-8c66-3f3eca54f951.mp4

**After**

https://user-images.githubusercontent.com/47562075/201494847-68534d70-82bd-44d0-88e6-a53f52df9509.mp4

Closes #2610

### CODE REVIEW

**Review by JulianaSau — CHANGES_REQUESTED** [review]
All in all, these are just minor fixes. Your code looks good and the functionality works great

File: packages/ui/src/message-room-details-dialog/MessageRoomDetailsDialog.jsx [hunk 1]
```diff
@@ -51,6 +51,7 @@ function MessageRoomDetailsDialog({
     setShowLeaveChannelModal(!showLeaveChannelModal);
   const toggleArchiveChannel = () => setShowArchiveChannel(!showArchiveChannel);
 
+  console.log(config, "from MessageRoomDetailsDialog");
```
[the remark this answers is not in the document]
Comment (OkoloJohnbosco) [reply packages/ui/src/message-room-details-dialog/MessageRoomDetailsDialog.jsx]: cleared

File: packages/ui/src/message-room-details-dialog/MessageRoomDetailsDialog.jsx [hunk 2]
```diff
@@ -273,11 +276,27 @@ function MembersPanel({ config }) {
...
+        });
+        setUserList(checkedUsers);
+        setisLoading(false);
       })
-      .catch(/*e => console.log("Organization not returning members", e)*/);
-    setisLoading(true);
-  }, []);
+      .catch(() => {
+        setisLoading(false);
+        /*e => console.log("Organization not returning members", e)*/
```
[the remark this answers is not in the document]
Comment (JulianaSau) [reply packages/ui/src/message-room-details-dialog/MessageRoomDetailsDialog.jsx:295]: Is your functionality using the users from the backend or just the dummy members only?

File: .vscode/settings.json [hunk 3]
```diff
@@ -4,11 +4,5 @@
     "**/src/**/*.js": "javascriptreact",
     "**/src/**/*.jsx": "javascriptreact",
     "**/.husky/": "bash"
-  },
```
Comment (JulianaSau) [inline .vscode/settings.json]: You can also just do away with these modifications on this file.

**Review by JulianaSau — CHANGES_REQUESTED** [review]

**Comment by OkoloJohnbosco** [comment]
It is using the users from the backend. the email on the demo is actually
mine

File: .vscode/settings.json (hunk 3 above)
Comment (OkoloJohnbosco) [reply .vscode/settings.json]: Done

**Review by JulianaSau — APPROVED** [review]

**Comment by JulianaSau** [comment]
@OkoloJohnbosco, pull changes from the dev branch and push to your remote repository. Then I'll merge your pr

**Comment by OkoloJohnbosco** [comment]
> @OkoloJohnbosco, pull changes from the dev branch and push to your remote repository. Then I'll merge your pr

ok boss

**Pull request merged**